forest/src/cli/sync_cmd.rs

```diff
@@ -0,0 +1,198 @@
+                        println!("\tStage: {}", active_sync.stage());
+                        println!("\tHeight: {}", active_sync.epoch);
+                        if let Some(end_time) = active_sync.end {
+                            if let Some(start_time) = active_sync.start {
+                                let zero_time = get_naive_time_zero();
+
+                                if end_time == zero_time {
+                                    if start_time != zero_time {
+                                        let time_now = get_naive_time_now();
+                                        println!(
+                                            "\tElapsed: {:?}\n",
+                                            time_now.signed_duration_since(start_time)
+                                        );
+                                    }
+                                } else {
+                                    println!(
+                                        "\tElapsed: {:?}\n",
+                                        end_time.signed_duration_since(start_time)
+                                    );
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+
+            SyncCommand::Wait {} => {
+                loop {
+                    // If not done syncing or runs into a error stop waiting
+                    if sync_wait(&mut client).await.unwrap_or(true) {
+                        break;
+                    }
+                }
+            }
+
+            SyncCommand::MarkBad { block_cid } => {
+                let response = mark_bad(&mut client, block_cid.clone()).await;
+                if response.is_ok() {
+                    println!("Successfully marked block {} as bad", block_cid);
+                } else {
+                    println!(
+                        "Failed to mark block {} as bad, error is {} ",
+                        block_cid,
+                        stringify_rpc_err(response.unwrap_err())
+                    );
+                }
+            }
+
+            SyncCommand::CheckBad { block_cid } => {
+                let response = check_bad(&mut client, block_cid.clone()).await;
+                if let Ok(reason) = response {
+                    println!("Block {} is \"{}\"", block_cid, reason);
+                } else {
+                    println!("Failed to check if block {} is bad", block_cid);
+                }
+            }
+            SyncCommand::Submit { gossip_block } => {
+                let response = submit_block(&mut client, gossip_block).await;
+                if response.is_ok() {
+                    println!("Successfully submitted block");
+                } else {
+                    println!(
+                        "Did not submit block because {:#?}",
+                        stringify_rpc_err(response.unwrap_err())
+                    );
+                }
```

Comment:
instead of doing `if ok {..} else {}` statements like this, you should match on the result to be able to log the error reason (and without unwrapping the error. (check-bad has no error message attached)